www/src/pages/editor/EditDownload.tsx

```diff
@@ -0,0 +1,32 @@
+import { DownloadOutlined } from "@ant-design/icons";
+import { Button } from "antd";
+import { SERVER } from "env";
+import { downloadURI } from "misc";
+import { NormSearchArgs } from "./SearchBar";
+import { observer } from "mobx-react-lite";
+
+interface DownloadButtonProp {
+  normSearchArgs: NormSearchArgs;
+}
+
+export const DownloadButton: React.FC<DownloadButtonProp> = observer(({ normSearchArgs }) => {
+  const handleDownload = () => {
+    let url: string;
+    const queryParams = new URLSearchParams();
+    queryParams.append("format", "csv");
+
+    if (normSearchArgs.commodity !== undefined) queryParams.append("commodity", normSearchArgs.commodity.id);
+    if (normSearchArgs.depositType !== undefined) queryParams.append("depositType", normSearchArgs.depositType.id);
+    if (normSearchArgs.country !== undefined) queryParams.append("country", normSearchArgs.country.id);
+    if (normSearchArgs.stateOrProvince !== undefined) queryParams.append("stateOrProvince", normSearchArgs.stateOrProvince.id);
```

Comment:
Checkout the API: https://minmod.isi.edu/api/v1/docs#/mineral_sites/dedup_mineral_sites_v2_api_v1_dedup_mineral_sites_get

it's snakecase, not camel case.